nginx.conf

```
@@ -2,6 +2,38 @@ upstream backend {
     server unix:/var/www/gunicorn.sock fail_timeout=0;
 }
 
+# Catch all, 404 in case a subdomain record exists, but we don't have the server block
+server {
+    server_name default_server;
+    listen 80;
+
+    root /var/www/html;
+
+    access_log /var/www/logs/csss-site-backend/nginx-access.log;
+    error_log /var/www/logs/csss-site-backend/nginx-error.log;
+
+    location / {
+        charset utf-8;
+        try_files 404.html;
```

Wouldn't this be? I think that **try_files** expects a fallback (=404, etc).
```suggestion
        try_files /404.html =404;
```